packages/config/adders/official.ts

```diff
@@ -4,8 +4,7 @@ export const adderCategories: AdderCategories = {
 	codeQuality: ['prettier', 'eslint'],
 	css: ['tailwindcss', 'bulma', 'bootstrap'],
 	db: ['drizzle'],
-	markdown: ['mdsvex'],
-	tools: ['storybook', 'routify'],
+	tools: ['storybook', 'routify', 'mdsvex'],
```

Comment:
ordering in terms of popularity
```suggestion
	tools: ['storybook', 'mdsvex', 'routify'],
```